include/zephyr/misc/lorem_ipsum.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2015 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_MISC_LOREM_IPSUM_H_
+#define ZEPHYR_MISC_LOREM_IPSUM_H_
+
+#define LOREM_IPSUM_SHORT                                                                          \
+	"Lorem ipsum dolor sit amet, consectetur adipiscing elit, "                                \
+	"sed do eiusmod tempor incididunt ut labore et dolore magna "                              \
+	"aliqua. Ut enim ad minim veniam, quis nostrud exercitation "                              \
+	"ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis "                           \
+	"aute irure dolor in reprehenderit in voluptate velit esse "                               \
+	"cillum dolore eu fugiat nulla pariatur. Excepteur sint "                                  \
+	"occaecat cupidatat non proident, sunt in culpa qui officia "                              \
+	"deserunt mollit anim id est laborum."
+
+#define LOREM_IPSUM                                                                                \
```

Comment:
I suggest adding this comment here from the networking sources:
```
/* Generated by http://www.lipsum.com/
 * 2 paragraphs, 179 words, 1160 bytes of Lorem Ipsum
 */
```
The length of the text was specifically set like this for networking purposes, although I am no longer sure about the length as it could have been longer to fit the IPv6 MTU. Anyway, if we add the comment, then the `\n` needs to be added here too (now it is only left in the net sources), otherwise the length is not exactly 1160 bytes.

Reply:
That's how I had it originally, so now I have an excuse to go back to that :-)

Reply:
@jukkar - the original snippet from `echo-client.c` is actually 1162 chars (not counting the terminating `\0` character) and 178 words 🤷‍♂️

Reply:
Ok, probably some extra `\n` was added after generating the lipsum. Dunno what the size should be, it has been so many years that reasoning for the size is lost :disappointed:

Reply:
We can fix it :-)

I did notice that one of the longer instances was 1210 bytes, which to me says it would intentionally be split over multiple IPv6 MTU.

Should I add that as `LOREM_IPSUM_LONG`? I've added `BUILD_ASSERT()` as well in my next revision (and a copyright :P)

Reply:
I doubt we need three different sizes. Perhaps what we have now, the short one and the "normal" one is ok. Just need to document what the size is, we can just pick a size, and probably either the 1160 or 1210 is just fine.